src/main/java/org/jenkinsci/test/acceptance/po/SlavesMixIn.java

```diff
@@ -54,4 +54,12 @@ private String remoteFs(String name) {
 
         return base + name;
     }
+
+    /**
+     * Get the master node of jenkins.
+     * @return The master node.
+     */
+    public DumbSlave getMaster() {
```

Comment:
All node related configuration options for master should be on the system config page.